extras/printschema.py

```diff
@@ -0,0 +1,45 @@
+import copy
```

Comment:
This file seems tangential to the rest of the diff. Could you pull it out and put it in a separate PR?

Also, on a side-note. My usual way of pretty-printing here is to use the [`json` npm package](https://www.npmjs.com/package/json):

```
$ echo '{"hi":"there"}' | genson | json -i
{ required: [ 'hi' ],
  type: 'object',
  properties: { hi: { type: 'string' } } }
```